### DISCUSSION TITLE
Different rendering for entrances

### DISCUSSION THREAD

**Discussion by hahaslav** [organicmaps/organicmaps#478] [Ideas]
#### Now Organic Maps render all entrances as a grey dot, but it doesn't show importance or type of entrance:
![Screenshot_20210602-113029](https://user-images.githubusercontent.com/65510612/120450609-416ce400-c399-11eb-9fae-f43f4480dc73.png)
#### It may confuse users. I suggest changing the look of some types of entrances:
![new](https://user-images.githubusercontent.com/65510612/120451207-e5568f80-c399-11eb-860b-6cae6188ece7.png)

Here I show an idea of the new look of some entrances, but they are not final, so we can discuss their new look. The `entrance=main` now looks bigger than the common entrance because of the circle around the dot. The `entrance=service` now looks like a small circle. Now users may guess which entrance of the building is main. Also, we can tell them the type of entrances in the description.

Furthermore, the reason for my suggestion is that 20.75% of entrances in OSM are `entrance=main` ([source](https://taginfo.openstreetmap.org/keys/?key=entrance#values)), so we can make the map looks more detailed by implementing this idea. `entrance=service` is used by 1.6% of entrances, but this type is still important because sometimes users need to go to the back door of the building.

###### Yes, it is poor explanation. Anyway, you are welcome to discuss this idea.

**Comment by biodranik** [comment]
What are other entrance types?
It would be better to draw recognized doors icons instead of unclear dots.

**Reply by hahaslav** [reply]
The main types of entrances described [here](https://wiki.openstreetmap.org/wiki/Key:entrance#Tagging).
In my opinion, it is good idea to draw doors instead of dots. But it should fit in the current style of the map.

**Reply by biodranik** [reply]
I suggest to choose some types as a "main" entrance and draw a clearly recognisable, half-open door icon. It should be smaller than any other icons to avoid map pollution.
For service and other exit types draw an even smaller closed door.

It would be great if you can find or draw some open source icons and integrate them into OM styles :)

**Reply by biodranik** [reply]
I suggest to choose some types as a "main" entrance and draw a clearly recognisable, half-open door icon. It should be smaller than any other icons to avoid map pollution.
For service and other exit types draw an even smaller closed door.

It would be great if you can find or draw some open source icons and integrate them into OM styles :)

**Reply by biodranik** [reply]
I suggest to choose some types as a "main" entrance and draw a clearly recognisable, half-open door icon. It should be smaller than any other icons to avoid map pollution.
For service and other exit types draw an even smaller closed door.

It would be great if you can find or draw some open source icons and integrate them into OM styles :)
